src/header/header.scss

```diff
@@ -190,11 +144,11 @@
         @include themed {
           background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
         }
+      }
 
-        &:not(:focus-visible):not(:hover) {
-          @include themed {
-            background-color: t(iui-color-background-1);
-          }
+      &:not(:focus-visible):not(:hover) {
+        @include themed {
+          background-color: t(iui-color-background-1);
         }
       }
     }
```

Comment:
Could not comment there, but naming of `iui-header-button-icon`, which is under `iui-header-logo` seems weird..

Reply:
Tried to cleanup the mixin & class names.  We now have:  `.iui-header-brand-icon`, `.iui-header-breadcrumbs-button-icon`, & `.iui-button-icon`.